certified-connectors/bitskout/apiDefinition.swagger.json

```diff
@@ -177,87 +177,87 @@
 									"properties": {
```

Comment:
@saferoombiz Please fix the indentation. all files should be properly formatted.
![image](https://user-images.githubusercontent.com/97551789/230667046-d60047f5-c581-4513-bd9b-c3d8500a228e.png)